skills/declarative/extensions/Microsoft.Bot.Solutions.Extensions/Actions/GetContacts.cs

```diff
@@ -40,7 +42,8 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             var name = this.NameProperty.GetValue(dcState);
             var token = this.Token.GetValue(dcState);
 
-            var graphClient = GraphClient.GetAuthenticatedClient(token);
+            dc.Context.TurnState.TryGetValue(MockHttpRequestMiddleware.HttpMessageHandlerKey, out var httpHandler);
+            var graphClient = GraphClient.GetAuthenticatedClient(token, (HttpMessageHandler)httpHandler);
```

Comment:
Will GraphClient.GetAuthenticatedClient create a new one if httpHandler is null?

Reply:
yes. the default value of httpHandler property is null